srcpkgs/poppler/template

```diff
@@ -3,13 +3,14 @@
 # THIS PKG MUST BE SYNCHRONIZED WITH "srcpkgs/poppler-qt5".
 #
 pkgname=poppler
-version=21.07.0
+version=22.03.0
```

Comment:
Can we split this to a separate PR? This will need some revbumps.

Reply:
Sure.